packages/wallet/test/persistence/pouchdbStores.test.ts

```diff
@@ -46,7 +47,8 @@ describe('pouchdbStores', () => {
   });
 
   describe('PouchdbCollectionStore', () => {
-    const createStore = () => new PouchdbCollectionStore<DocType>(dbName, ({ __unsupportedKey }) => __unsupportedKey);
+    const createStore = () =>
+      new PouchdbCollectionStore<DocType>(dbName, dummyLogger, ({ __unsupportedKey }) => __unsupportedKey);
```

Comment:
That's a bit awkward too. `undefined` is better than needing to import the dummyLogger. Maybe revert, and we can come back to this in a params logger consistency check?

Reply:
See [cb36cbb](https://github.com/input-output-hk/cardano-js-sdk/pull/263/commits/cb36cbbbb8c19e41d034092ddc015cf8417a456d)